e2e/drone.sh

```diff
@@ -43,3 +43,58 @@ echo "Checking that kontena-lens is running:"
 
 # Rerun up to confirm that non-initial run goes through
 timeout 300 pharos up -y -c e2e/digitalocean/cluster.yml --tf-json e2e/digitalocean/tf.json || exit $?
+
+if [ ! -f e2e/digitalocean/worker.json ]; then
+  echo "Worker JSON not found, skipping."
+  exit 0
+fi
+
+echo "Setting up 'pharos worker up'.."
+
+apt-get update && apt-get install -y jq || exit $?
+
+pharos_worker_host=$(jq ".address[0]" e2e/digitalocean/worker.json | sed 's/"//g')
+
+if [ "${pharos_worker_host}" = "" ]; then
+  echo "Couldn't get pharos worker host from worker.json"
+  exit 1
+fi
+
+ssh_key="e2e/digitalocean/ssh_key.pem"
+ssh_userhost="root@${pharos_worker_host}"
+
+scp -o StrictHostKeyChecking=no -i "${ssh_key}" pharos-cluster*.gem "${ssh_userhost}:"
+ssh -o StrictHostKeyChecking=no -i "${ssh_key}" "${ssh_userhost}" -- "sudo http_proxy=http://10.133.37.156:8888 apt-get update"
```

Comment:
Multiple `ssh` lines with same envs look really messy. Could we combine these to single `ssh` call?